edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/RouteConfiguration.cs

```diff
@@ -22,8 +23,15 @@ public RouteConfiguration(string route)
         public RouteConfiguration(string route, uint priority, uint timeToLiveSecs)
         {
             this.Route = Preconditions.CheckNonWhiteSpace(route, nameof(route));
-            this.Priority = priority;
             this.TimeToLiveSecs = timeToLiveSecs;
+
+            // Verify the route, this must be either between [0-9], or the default value
+            if ((priority < 0 || priority > 9) && priority != RouteFactory.DefaultPriority)

```

Comment:
Do we have tests for this?

Reply:
What happens when I set a route to 20? Does EH log the error and continue chugging along? Gets default priority?

Reply:
no, the exception is caught and sent up as reported status by the Agent